packages/tools/examples/videoSegmentation/index.ts

```diff
@@ -0,0 +1,403 @@
+import {
+  Enums,
+  RenderingEngine,
+  imageLoader,
+  utilities as csUtils,
+} from '@cornerstonejs/core';
+import * as cornerstone from '@cornerstonejs/core';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+import {
+  createImageIdsAndCacheMetaData,
+  initDemo,
+  addDropdownToToolbar,
+  setTitleAndDescription,
+  addButtonToToolbar,
+  getLocalUrl,
+  addManipulationBindings,
+  addVideoTime,
+} from '../../../../utils/demo/helpers';
+import { fillStackSegmentationWithMockData } from '../../../../utils/test/testUtils';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  ToolGroupManager,
+  SegmentationDisplayTool,
+  Enums: csToolsEnums,
+  RectangleScissorsTool,
+  CircleScissorsTool,
+  BrushTool,
+  PaintFillTool,
+  segmentation,
+  utilities: cstUtils,
+} = cornerstoneTools;
+
+const { MouseBindings } = csToolsEnums;
+const { ViewportType } = Enums;
+const { segmentation: segmentationUtils } = cstUtils;
+
+// Define a unique id for the volume
+let renderingEngine;
+const renderingEngineId = 'myRenderingEngine';
+const viewportId = 'VIDEO_VIEWPORT';
+const toolGroupId = 'TOOL_GROUP_ID';
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Segmentation in VideoViewport',
+  'Here we demonstrate how to render a segmentation in a VideoViewport'
```

Comment:
Render a label map segmentation. We are advocating for the idea of separating segmentation from label maps.